payments/models.py

```diff
@@ -0,0 +1,83 @@
+
+import uuid
+from django.core.urlresolvers import reverse
+from django.db import models
+import getpaid
+
+
+CURRENCIES = (
+    ('DKK', 'DKK'),
+    ('PLN', 'PLN'),
+    ('GBP', 'GBP'),
+)
+
+
+class OrderManager(models.Manager):
+
+    def create_for_fee(self, user, fee):
```

Comment:
The way I see this is similar to creating other instance of any model: `MyModel.objects.create(...)`,
similarly to User manager, and this is why I though it would be good to have it in manager.